src/main/Resources/traffic.csv

```diff
@@ -0,0 +1,13 @@
+Car, WantsToDrive

```

Comment:
I don't know whether there was a particular reason to use lower case back in the old days. I just know it as a convention. A consequence of that is that Maven uses certain paths (like `src/main/resources`) by default and you'd have to actually configure it to use different values.